src/lib.rs

```diff
@@ -558,6 +559,22 @@ fn projection_name_from_expr(e: Expr) -> String {
     }
 }
 
+#[wasm_bindgen]
+extern { pub fn alert(s: &str); }
+
+#[wasm_bindgen]
+pub fn extract_table_names(sql: String) -> JsValue {
```

Comment:
@Anks I'm not sure which is the best option here monadic error handling or raising alerts.

Reply:
For now, I have just made the function return a string (in case of error) or an array of strings in case of success. Not a great interface, but it allows the caller to distinguish between parsing failure and empty state